src/main/java/edu/usf/sas/pal/muser/ProcessorMain.java

```diff
@@ -45,12 +40,14 @@ public static void main(String[] args) {
             if (cmd.hasOption(ProgramOptions.USER_ID)) {
                 programOptions.setUserId(cmd.getOptionValue(ProgramOptions.USER_ID));
             }
+            if (cmd.hasOption(ProgramOptions.START_DATE) && cmd.hasOption(ProgramOptions.END_DATE)) {
```

Comment:
If only one of the two options here is provided we should output an error message and abort.